includes/functions.php

```diff
@@ -2285,9 +2285,13 @@ function pmpro_getLevelAtCheckout( $level_id = null, $discount_code = null ) {
 		if ( empty( $level_id ) ) {
 			$all_levels = pmpro_getAllLevels( false, false );
 
-			// Get lowest level ID.
-			$default_level =  min( array_keys( $all_levels ) );
-
+			if ( ! empty( $all_levels ) ) {
+				// Get lowest level ID.
+				$default_level =  min( array_keys( $all_levels ) );
```

Comment:
Should the intval be called anymore below, should it be called here?

Reply:
Nice catch. I'll move the intval up.

Reply:
Actually, I might leave it where it is. As far as I know, nothing relies on that default_level value being null, 0, or false specifically, just that it's empty. For the sake of making the smallest possible change here unless we have a reason to, I'm going to keep the intval where it is.